variables.tf

```diff
@@ -70,7 +70,7 @@ variable "infra_count" {
 
 variable "infra_flavor" {
   type        = string
-  default     = "plus-16"
+  default     = "plus-32-4"
```

Comment:
Default infra node flavor assumed no logging stack until now, so we should keep it as `plus-16-4`, wdyt @corvus-ch?